projects/extension/sql/incremental/008-fix-broken-owner-check.sql

```diff
@@ -0,0 +1 @@
+drop function if exists ai._vectorizer_create_dependencies(integer);
```

Comment:
Wait why do we need this

Reply:
doh - we don't. It was a stray leftover from before I removed some other bits.

Reply:
fixed.